web/src/app/modules/shared/components/presentation/signpost/signpost.component.ts

```diff
@@ -0,0 +1,44 @@
+import { ChangeDetectorRef, Component, SecurityContext } from '@angular/core';
+import { AbstractViewComponent } from '../../abstract-view/abstract-view.component';
+import { SignpostView, View } from '../../../models/content';
+import { parse } from 'marked';
+import { DomSanitizer } from '@angular/platform-browser';
+
+@Component({
+  selector: 'app-signpost',
+  templateUrl: './signpost.component.html',
+  styleUrls: ['./signpost.component.scss'],
+})
+export class SignpostComponent extends AbstractViewComponent<SignpostView> {
+  message: string;
+  position: string;
+  component: View;
+  trigger: View;
+
+  directions = {
+    column: false,
+  };
+
+  order = {
+    reverse: false,
+  };
+
+  constructor(
+    private cdr: ChangeDetectorRef,
+    private readonly sanitizer: DomSanitizer
+  ) {
+    super();
+  }
+
+  update() {
+    this.directions.column = this.v.config.column || false;
+    this.order.reverse = this.v.config.reverse || false;
```

Comment:
Aren't these false by default?

Reply:
Yes! I will fix this

Reply:
Actually, I think this could fail on storybook if values are not set properly